routemaster/client.rb

```diff
@@ -9,17 +11,21 @@
 
 module Routemaster
   class Client
-    
+
     def initialize(options = {})
-      @_url = _assert_valid_url(options[:url])
-      @_uuid = options[:uuid]
-      @_timeout = options.fetch(:timeout, 1)
-      @_verify_ssl = options.fetch(:verify_ssl, true)
+      @_options = options.tap do |o|
```

Comment:
As far as i can see the `@_options` has local scope in this method so probably we can use just a local var here.